### ISSUE TITLE
OPcache not properly configured out-of-the-box according to the "occ setupcheck"

### ISSUE THREAD

**Issue by AlphaCraft9658** [nextcloud/all-in-one#4616]
### Steps to reproduce
This may only work on out-of-the-box setups without direct manual modifications to configuration files.
1. Enter host shell
2. Execute the following command: `docker exec --user www-data nextcloud-aio-nextcloud php occ setupcheck` (running with sudo may be required)
3. Read the output of the command. It should say that the OPcache is not properly configured.

### Expected behavior 
All setupchecks should be successfull and opcache should be properly configured out-of-the-box.

### Actual behavior 
The setupcheck tells that the opcache is not properly configured.

### Host OS 
Ubuntu Server 20.04 with Docker-CE

#### Nextcloud AIO version 
v8.2.1

#### Current channel 
latest

#### Other valuable info 
The section of the `setupcheck` command's output telling about the improper OPcache configuration:
```
✗ PHP opcache: The PHP OPcache module is not properly configured. OPcache is not working as it should, opcache_get_status() returns false, please check configuration.
The maximum number of OPcache keys is nearly exceeded. To assure that all scripts can be kept in the cache, it is recommended to apply "opcache.max_accelerated_files" to your PHP configuration with a value higher than "10000".
The OPcache buffer is nearly full. To assure that all scripts can be hold in cache, it is recommended to apply "opcache.memory_consumption" to your PHP configuration with a value higher than "256".
The OPcache interned strings buffer is nearly full. To assure that repeating strings can be effectively cached, it is recommended to apply "opcache.interned_strings_buffer" to your PHP configuration with a value higher than "64"..
```

**Comment by Procsiab** [comment]
I am experiencing the exact same issue after updating Nextcloud from 28.0.5 to 29.0.0

**Comment by AlphaCraft9658** [comment]
> I am experiencing the exact same issue after updating Nextcloud from 28.0.5 to 29.0.0

I restored a backup after manually updating to Nextcloud 29, but this still happens on Nextcloud 28.

**Comment by Procsiab** [comment]
What did you restore from backup? I am performing a restore right now of both the database volume and the app/data volume

**Comment by AlphaCraft9658** [comment]
> What did you restore from backup? I am performing a restore right now of both the database volume and the app/data volume

I had a full backup, which should include all containers.

**Comment by AlphaCraft9658** [comment]
> > What did you restore from backup? I am performing a restore right now of both the database volume and the app/data volume
> 
> I had a full backup, which should include all containers.

I checked. It does include containers, as my Nextcloud is back on version 28. It should include **everything** related to Nextcloud AIO, including user data.

**Comment by Procsiab** [comment]
I'm on the same boat apparently

**Comment by Procsiab** [comment]
It seems that this was present before Nextcloud 29 (or Hub 8): after logging in on another server of mine which is running Nextcloud 28.0.4 I can see the exact same error of your first comment; however, in this case the opcache error is not shown from the admin web interface but only from running the setupchecks from CLI, so I did not notice it until in Nextcloud 29 it was signaled over there.

**Comment by AlphaCraft9658** [comment]
> It seems that this was present before Nextcloud 29 (or Hub 8): after logging in on another server of mine which is running Nextcloud 28.0.4 I can see the exact same error of your first comment; however, in this case the opcache error is not shown from the admin web interface but only from running the setupchecks from CLI, so I did not notice it until in Nextcloud 29 it was signaled over there.

Yes. In Nextcloud 28 you need to run the setupcheck, which shows the error.

**Comment by szaimen** [comment]
Hi, can you please open an issue about this here?
https://github.com/nextcloud/server
Thanks!

**Comment by AlphaCraft9658** [comment]
> Hi, can you please open an issue about this here?
> https://github.com/nextcloud/server
> Thanks!

I figured it could be an issue with the default configuration AIO provides. Could also not be the case.

**Comment by szaimen** [comment]
No, I think the check does not take into account that the web server might be using different php.ini files than the cli...
